Orange/canvas/gui/utils.py

```diff
@@ -21,16 +22,46 @@
 QWIDGETSIZE_MAX = ((1 << 24) - 1)
 
 
-@contextmanager
-def updates_disabled(widget):
-    """Disable QWidget updates (using QWidget.setUpdatesEnabled)
+# Serves as a decorator and context manager, hence a function-like name
+# pylint: disable=invalid-name
```

Comment:
Put on the next line, otherwise it disables for the whole indentation level (i.e. module).